circle.yml

```diff
@@ -0,0 +1,18 @@
+dependencies: 
+  cache_directories: 
+    - ~/.cache/pip
+    - ~/wheelhouse
+  post: 
+    - .circle/services.sh
+    - ".circle/compose.sh run wheezy /bin/true"
+    - ".circle/compose.sh run wheezy"
```

Comment:
One `compose.sh` script doing 3 different things and launched 3 times is confusing.

The usual dependencies CI config that is more clear for someone who looks into it: 
```yaml
dependencies:
  cache_directories:
    - ~/.cache/pip
    - ~/wheelhouse
  override:
    - sudo pip install wheel
    - sudo pip install docker-compose
```

Easier to follow the logic, instead of digging into many bash scripts. Also once you divide them as separate commands - CircleCI shows time for every command, what's good to know for debugging.

That's probably the reason why you can see something like: 
https://github.com/ruby/ruby/blob/trunk/.travis.yml